### DISCUSSION TITLE
Multiple Response for View Configuration

### DISCUSSION THREAD

**Discussion by Siburanjan** [zwave-js/zwave-js#7108] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

When we are trying to view configuration for a device then sometimes the Zwave Server is responding twice/multiple times

We expect it to respond only once for a API so that handling it in our firmware end is smoother

Coud you please help us out resolving this any way/fix?

### Software versions

Driver (node-zwave-js): 11.14.3

Z-Wave JS UI: 1.0.0

### Device information

Manufacturer: Aeotec(AEON LAB)
Model name: Smart Switch
"MID": "0086"
"PID": "0060"
"PTID": "0203"
Node ID: 92

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

file:///home/siburanjan/SIBU/Edge_HUB/hub%20Isuue/logs/zwavejs_2024-08-14.log
[node_92.json](https://github.com/user-attachments/files/16612932/node_92.json)

[a comment by an automated account is not shown]

**Reply by Siburanjan** [reply]
Thanks for the comment the file was not uploaded properly.
I am uploading the correct log file along with this reply

[zwavejs_2024-08-14.log](https://github.com/user-attachments/files/16613057/zwavejs_2024-08-14.log)

**Comment by AlCalzone** [comment]
Can you be a bit more specific about the exact situation where you got multiple reports? There's quite a bit of traffic between Z-Wave JS and Node 092 in that log.

FWIW, this could also be caused by connectivity issues. For example this situation where Node 109 sends the same report multiple times:
```
2024-08-14T01:38:14.917Z DRIVER « [Node 109] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -89 dBm
                                  └─[ConfigurationCCReport]
                                      parameter #: 85
                                      value size:  1
                                      value:       5
2024-08-14T01:38:14.924Z CNTRLR   [Node 109] The node is now alive.
...
2024-08-14T01:38:15.242Z DRIVER « [Node 109] [REQ] [BridgeApplicationCommand]
                                  │ type:        broadcast
                                  │ target node: 255
                                  │ RSSI:        -89 dBm
                                  └─[ConfigurationCCReport]
                                      parameter #: 85
                                      value size:  1
                                      value:       5
```

The fact that it uses a broadcast frame usually means it tried to reach the controller multiple times before without success.

When there are connectivity issues, there are a few troubleshooting steps we always recommend: https://zwave-js.github.io/node-zwave-js/#/troubleshooting/first-steps
Might be worth taking a look at those.
